python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1613,6 +1615,11 @@ def getTargetList(parsedCmd, **kwargs):
                 refList1, refList2 in zip(parsedCmd.id.refList, idParser.refList)]
 
 
+def patch2Int(patch):
+    assert len(patch) == 3
```

Comment:
Where does this limitation come from? I work on tracts that have a patch like `15,20`, which would violate this.

I think you should do:

```
def patchToInt(patch, maxPatches):
    """docstring"""
    elements = patch.split(",")
    assert len(elements) == 2
    return maxPatches*int(elements[0]) + int(elements[1])
```

(You can get `maxPatches` from the skyMap, or make it a config parameter if you're desperate. Hard-coding a value will cause problems later.)

Reply:
Definitely. Good point.